docs/content/preview/integrations/camunda.md

```diff
@@ -0,0 +1,173 @@
+- Java JDK 1.8+
+- NodeJS >= v10 
+- [Camunda Modeler](https://camunda.com/download/modeler).
+
+## Configure Camunda Platform 7
+- Download Camunda Platform 7 from [here](https://camunda.com/download/) and unzip it.
+- Locate the `spring.datasource` section in the below files
+  - `camunda-bpm-run-7.17.0/configuration/default.yml`
+  - `camunda-bpm-run-7.17.0/configuration/production.yml`
+- Modify this section with the configuration given below which uses standard Postgresql JDBC driver.
+```yml
+# datasource configuration is required
+spring.datasource:
+ url: jdbc:postgresql://localhost:5433/yugabyte
+ driver-class-name: org.postgresql.Driver
+ username: yugabyte
+ password: yugabyte
+```
+Change the `url` to point to the YugabyteDB cluster you started.
+- Download the Postgres JDBC driver [jar](https://jdbc.postgresql.org/download/postgresql-42.3.5.jar) and place it in `camunda-bpm-run-7.17.0/configuration/userlib` directory. 
+- Alternatively for using YugabyteDB JDBC driver use the below configuration.
+```yml
+# datasource configuration is required
+spring.datasource:
+ url: jdbc:yugabytedb://localhost:5433/yugabyte
+ driver-class-name: com.yugabyte.Driver
+ username: yugabyte
+ password: yugabyte
+```
+- Download the YugabyteDB JDBC driver [jar](https://repo1.maven.org/maven2/com/yugabyte/jdbc-yugabytedb/42.3.5-yb-1/jdbc-yugabytedb-42.3.5-yb-1.jar) and place it in `camunda-bpm-run-7.17.0/configuration/userlib` directory. You can read more about YugabyteDB JDBC driver [here](https://docs.yugabyte.com/preview/integrations/jdbc-driver/).
+
+- Run the Camunda Platform server using `./start.sh` for Linux/MacOS systems or `./start.bat` for Windows System.
+
+## Verify the integration
+Once server is started you will be able to see a few tables getting created in the database. 
+
+To do so
+- Login to the database using `./bin/ysqsh`
+- Use `\d` to see the list of tables.
+- Verify that a list of tables with the prefix `ACT_` are created.
+  More details regarding the tables created are mentioned [here](https://docs.camunda.org/manual/7.16/user-guide/process-engine/database/database-schema/).
+
+## Hello World Application using Camunda
+We are going to use Camunda Modeler to design and deploy a simple BPMN for charging the cards. 
+
+This example is taken from the [quick-start example](https://docs.camunda.org/get-started/quick-start/) in Camunda Docs.
+
+We are going to need three events:
+- Start Event   
+- Service Event
+- End Event
+
+For Service Event we are going to need an external task worker, for that purpose we are going to use `Nodejs`.
+Download the Camunda Modeler from [here](https://camunda.com/download/modeler) and launch it.
+
+- Create a new BPMN diagram by clicking File > New File > BPMN Diagram.
```

Comment:
Don't link to external images. This screenshot doesn't add much value anyway, so maybe we can just remove it.